content/en/developer-hub/_index.html

```diff
@@ -0,0 +1,93 @@
+---
+title: Developer Hub
+linkTitle: Overview
+main_menu: false
+weight: 1
+menu:
+  main:
+    name: "Developer Hub"
+    weight: 10
+cascade:
+  type: docs
+hide_feedback: true
+hide_readingtime: true
+dev_hub: true
+---
+
+<div class="doc-overview-grid container">
+
+  <div class="row mb-3 justify-content-center">
+    <div class="col-auto mb-2">
+      <img src="/images/localstack.svg" alt="LocalStack logo" width="75px">
+    </div>
+    <div class="col-auto">
+      <p class="lead text-center">
+        Welcome to the LocalStack Developer Hub!<br>
+        Explore and learn about LocalStack with concrete examples.
+      </p>
+    </div>
+  </div>
+
+  <div class="row mb-4">
+    <div class="col-sm-12 col-md-6">
+      <a class="card card-tutorials" href="{{< ref "tutorials" >}}">
+        <div class="card-body">
+          <h5 class="card-title">Tutorials</h5>
+          <p class="card-text">Read through our tutorials to learn more about LocalStack.</p>
+        </div>
+      </a>
+    </div>
+    <div class="col-sm-12 col-md-6">
+      <a class="card card-applications" href="{{< ref "integrations" >}}">
```

Comment:
This link seems to be wrong, I think it should lead to "applications". 😉 
Just browsed through it a bit, lookin' good! 🚀

Reply:
Thanks for the review :+1: 
Adjusted the link accordingly.